### PR TITLE
Update operations-and-monitoring.md

### PR DESCRIPTION
**Pull request by rekhanr**
Updated the section Device Groups, to add the link to the new feature and update responsibilities for both Microsoft and customer.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/service-description/operations-and-monitoring.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Categories | Microsoft will | Customer will
...
+Device Groups | - Implement and assign the membership of devices within Microsoft Managed Desktop groups.<br>- Use the Microsoft Managed Desktop groups to manage the assignment and release of configuration and updates to devices. | - Create a support request requesting information for a planned configuration change, including configuration details, scope, timeline, and other pertinent details for Microsoft to review.<br>- Only apply a change once Microsoft Managed Desktop Operations has assessed and advised.<br>- Only assign devices to any of Microsoft Managed Desktop group, following the steps described in [Assign devices to a deployment group](https://docs.microsoft.com/microsoft-365/managed-desktop/working-with-managed-desktop/assign-deployment-group?view=o365-worldwide).<br>- Only use the groups to assign corporate certificates for services such as VPN, Windows Hello for Business or email encryption, or corporate Wi-Fi profile configuration.<br>- Where co-management exists, explicitly exclude all Microsoft Managed Desktop groups when deploying the Configuration Manager client.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/operations-and-monitoring.md]:
```suggestion
Device Groups | - Implement and assign the membership of devices within Microsoft Managed Desktop groups.<br>- Use the Microsoft Managed Desktop groups to manage the assignment and release of configuration and updates to devices. | - Create a support request requesting information for a planned configuration change, including configuration details, scope, timeline, and other pertinent details for Microsoft to review.<br>- Only apply a change once Microsoft Managed Desktop Operations has assessed and advised.<br>- Only assign devices to any Microsoft Managed Desktop group following the steps described in [Assign devices to a deployment group](https://docs.microsoft.com/microsoft-365/managed-desktop/working-with-managed-desktop/assign-deployment-group?view=o365-worldwide).<br>- Only use the groups to assign corporate certificates for services such as VPN, Windows Hello for Business or email encryption, or corporate Wi-Fi profile configuration.<br>- Where co-management exists, explicitly exclude all Microsoft Managed Desktop groups when deploying the Configuration Manager client.
```

Comment (JohanFreelancer9) [reply microsoft-365/managed-desktop/service-description/operations-and-monitoring.md]: Please change the line "Only assign devices to any of Microsoft Managed Desktop group, following the steps..." to "Only assign devices to any Microsoft Managed Desktop group following the steps..." (delete the word "of" and the comma).

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rekhanr, kindly commit the suggested change. Many thanks.

**Comment by rekhanr** [comment]
@jaimeo FYI.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks

File: microsoft-365/managed-desktop/service-description/operations-and-monitoring.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Categories | Microsoft will | Customer will
...
+Device Groups | - Implement and assign the membership of devices within Microsoft Managed Desktop groups.<br>- Use the Microsoft Managed Desktop groups to manage the assignment and release of configuration and updates to devices. | - Create a support request requesting information for a planned configuration change, including configuration details, scope, timeline, and other pertinent details for Microsoft to review.<br>- Only apply a change once Microsoft Managed Desktop Operations has assessed and advised.<br>- Only assign devices to any Microsoft Managed Desktop group following the steps described in [Assign devices to a deployment group](https://docs.microsoft.com/microsoft-365/managed-desktop/working-with-managed-desktop/assign-deployment-group?view=o365-worldwide).<br>- Only use the groups to assign corporate certificates for services such as VPN, Windows Hello for Business or email encryption, or corporate Wi-Fi profile configuration.<br>- Where co-management exists, explicitly exclude all Microsoft Managed Desktop groups when deploying the Configuration Manager client.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/operations-and-monitoring.md]:
```suggestion
Device Groups | - Implement and assign the membership of devices within Microsoft Managed Desktop groups.<br>- Use the Microsoft Managed Desktop groups to manage the assignment and release of configuration and updates to devices. | - Create a support request requesting information for a planned configuration change, including configuration details, scope, timeline, and other pertinent details for Microsoft to review.<br>- Only apply a change once Microsoft Managed Desktop Operations has assessed and advised.<br>- Only assign devices to any Microsoft Managed Desktop group following the steps described in [Assign devices to a deployment group](/microsoft-365/managed-desktop/working-with-managed-desktop/assign-deployment-group?view=o365-worldwide).<br>- Only use the groups to assign corporate certificates for services such as VPN, Windows Hello for Business or email encryption, or corporate Wi-Fi profile configuration.<br>- Where co-management exists, explicitly exclude all Microsoft Managed Desktop groups when deploying the Configuration Manager client.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @rekhanr - please also change the link to a relative link as indicated. Should be (/microsoft-365/managed-desktop/working-with-managed-desktop/assign-deployment-group?view=o365-worldwide). Thank you.

**Comment by rekhanr** [comment]
Thank you. FYI @jaimeo .

**Comment by jaimeo** [comment]
@JohanFreelancer9 that is not the correct link format for links between articles in the same repo. I've corrected it.

**Pull request merged**